tests/src/Kernel/MessageTemplateCreateTrait.php

```diff
@@ -35,6 +35,20 @@ protected function createMessageTemplate($template, $label, $description, array
         'clear' => FALSE,
       ],
     ];
+
+    // If the $text array is simple text values, transform to text + format.
```

Comment:
In this case, we could require all callers to specify a `format`, but since this is a trait for making testing easier, I initially decided to support the legacy input of just an array of text strings:

```php
$text = [
  'text string one',
  'text string two',
];
```
as opposed to updating all callers to use the now more verbose input:

```php
$text = [
  [
    'value' => 'text string one',
   'format' => 'some_format',
  ],
  ...
];
```